### DISCUSSION TITLE
Add a BUILD.md or add instructions

### DISCUSSION THREAD

**Discussion by polymo1** [monkeytypegame/monkeytype#2842] [Ideas] [answered]
Hello! My school blocks monkeytype half of the week and I would like to host my own site with it for private use.

How would I go about running this?

**Comment by rizwanmustafa** [comment, marked as answer]
I recommend you give `CONTRIBUTING.md` a read. After you have set the project up, here are the remaining deployment steps.

### Frontend
For the frontend, you need to create a production build of the latest release of the monkeytype and serve it using a web server. You can do so by:
```bash
cd frontend
npm run build:live
cd public
npx server -p 5000
```

### Backend
For the backend, you need to transpile from typescript to javascript and execute the result. You can do so by:
```bash
cd backend
npm run clean
npm run start
```
